packages/web-components/src/card/card.spec.md

```diff
@@ -0,0 +1,93 @@
+# Slider
+
+## Component Description
+
+A card is a container that holds information and actions related to a single concept or object, like a document or a contact.
+
+## Design Spec
+
+[Link to Design Spec in Figma](https://www.figma.com/file/berhUBA6mJV9sCPpjgfKRj/Card?node-id=1520-2077&t=65kUw9Qr7VEhg1ra-11)
+
+## Engineering Spec
+
+### Inputs
+
+- @attr size: "small" "medium" "large" | "medium"
+- @attr orientation: "horizontal" "vertical" | "vertical"
+- @attr appearance: "filled" "filled-alternative" "outline" "subtle" | "filled"
+
+### Outputs
+
+None
+
+### Events
+
+- None yet
+
+### Slots
+
+- default: the focal point of the card, its body
+- header: The header area of a card, which represents a Fluent UI compliant card header
+- footer: The footer area of a card, which represents a Fluent UI compliant card footer. Uses a flex layout to organize actions the user can take with a card.
+- preview: Used inside of a card and allows the user to better visualize what the card's contents are
+
+### CSS Variables
+
+None
+
+### Interactions
+
+- None yet
+- Will be adding the ability to select a card via an internal checkbox that renders when the card is set to selectable
+
+## Accessibility
+
+- [x] Find the matching component through WCAG's patterns: https://www.w3.org/WAI/ARIA/apg/patterns/
+  - https://www.w3.org/WAI/ARIA/apg/patterns/slider/
+- [x] Are there any accessibility elements unique to this component?
+  - [x] No
+- [x] Does the component support 400% zoom?
+  - [x] Yes
+- [x] What keyboard behaviors does the component support?
+  - None yet, possibly some with selection
+
+## Preparation
+
```

Comment:
@procload I think the fluent card design guidelines should be here. did you review these as well? 

https://www.figma.com/file/8iZmitafCxS6LgKCCyDaFj/Card-templates?type=design&node-id=2254%3A57409&t=MAeL6uz4GuNizaRh-1